coreos-meta-translator/trans.py

```diff
@@ -102,6 +102,15 @@ def artifact(i):
                     "image": ami_dict["hvm"]
                 }
 
+        # GCP specific additions
+        if input_.get("gcp", None) is not None:
+            arch_dict["media"]["gcp"] = arch_dict["media"].get("gcp", {})
+            arch_dict["media"]["gcp"]["image"] = arch_dict["media"]["gcp"].get("image", {})
+            arch_dict["media"]["gcp"]["image"].update(input_.get("gcp", {}))
+            arch_dict["media"]["gcp"]["image"]["name"] = arch_dict["media"]["gcp"]["image"].pop("image", None)
```

Comment:
My suggestion is to make it a hard error so we don't silently continue if `image` is missing from `meta.json`:

```suggestion
            arch_dict["media"]["gcp"]["image"]["name"] = arch_dict["media"]["gcp"]["image"].pop("image")
```

Reply:
Oh I see, and the previous `pop("image")` too. Updating